### ISSUE TITLE
Is this repo dead or project dead?

### ISSUE THREAD

**Issue by ourboyfriend** [TIBCOSoftware/flogo#575]
Can't see the updates on this repo. But the community seems alive: https://github.com/project-flogo

**Comment by ayh20** [comment]
Yes, https://github.com/project-flogo is the live repo .. I guess this one is still around for backward compatibility reasons only.

**Comment by anborg** [comment]
Nice, well designed project, not sure why it was taken down. Has potential to shine.

**Comment by ayh20** [comment]
@anborg Not dead just moved to https://github.com/project-flogo
